Cargo.toml

```diff
@@ -19,6 +19,8 @@ data-encoding = "2.0.0"
 rand = "0.3.15"
 scan_fmt = "0.1.1"
 semver = "0.8.0"
+serde = "1"
+serde_derive = "1"
```

Comment:
Just FYI, these two dependencies were already being compiled as standard dependencies because they are required by other deps. So these are not actually new dependencies, they are now just "top-level" dependencies.

Reply:
Yep, `bson` already pulls those in. Thanks for the heads up!